bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/SatelModule.java

```diff
@@ -227,18 +240,23 @@ public boolean sendCommand(SatelCommand cmd, boolean force) {
 
     @Override
     public void incomingEvent(SatelEvent event) {
-        if (event instanceof IntegraVersionEvent) {
+        if (event instanceof ModuleVersionEvent) {
+            ModuleVersionEvent versionEvent = (ModuleVersionEvent) event;
+            this.extPayloadSupported = versionEvent.isExtPayloadSupported();
+            logger.info("Module version: {}.", versionEvent.getVersion());
```

Comment:
How often do these events occur? I would guess it's more a debug than an info. (also below)

Reply:
It happens once after connection is established for the first time. I don't think it is too verbose.